apps/chat/src/components/Chat/ConversationSettings.tsx

```diff
@@ -179,7 +182,7 @@ export const ConversationSettings = ({
         <div className="shrink bg-layer-2 px-3 py-4 md:px-6">
           <ConversationSettingsModel
             conversation={conversation}
-            modelId={model?.id}
+            modelId={model?.reference ?? recentModelsIds[0]}
```

Comment:
I suppose we always have reference? Maybe we should change type?

Reply:
but I don't have such code, the code looks like this

<ConversationSettingsModel
            conversation={conversation}
            modelId={model?.reference ?? recentModelsIds[0]}
            unavailableModelId={
              !model?.id && !isPseudoModel(modelId) ? modelId : undefined
            }
            onModelSelect={onSelectModel}
          />